projects/observability/src/shared/components/gauge/gauge.component.ts

```diff
@@ -104,11 +104,13 @@ export class GaugeComponent implements OnChanges {
   }
 
   private buildValueArc(radius: number, inputData: GaugeInputData): string {
+    const completeAngle = Math.PI / 2 + GaugeComponent.EXTRA_ARC_ANGLE;
+
     return this.buildArcGenerator()({
       innerRadius: radius - GaugeComponent.GAUGE_RING_WIDTH,
       outerRadius: radius,
-      startAngle: -Math.PI / 2 - GaugeComponent.EXTRA_ARC_ANGLE,
-      endAngle: -Math.PI / 2 - GaugeComponent.EXTRA_ARC_ANGLE + (inputData.value / inputData.maxValue) * Math.PI
+      startAngle: -completeAngle,
```

Comment:
Does this handle the first case described in the comment? If inputData.value is 0, we're doing `[-completeAngle, -completeAngle]` so there'd still be no arc

Reply:
Yes. We want no value arc, when the inputData.value is 0. In fact @anandtiwary mentioned that's the expected outcome as well

>case 1: inputData.value = 0, the end angle would be -Math.PI / 2 - GaugeComponent.EXTRA_ARC_ANGLE which is same as start angle. So the arc will be completely empty. Expected outcome.

Reply:
Ah I misread his comment - I thought our ux generally has a little arc for a 0-value gauge to show it's "emptyiness"? I'm pretty sure that's what we do in the linear case.

Reply:
hmm. fair point. I can use a value of 0.5, if the value is 0 to display the 0-value arc

Reply:
I think that makes sense - something like `percentFill = Math.min(0.01, inputData.value / inputData.maxValue)`

Reply:
I am not sure it looks any better to show a little arc for 0. 

Also, could we rename completeAngle to just startAngle?

Reply:
> Also, could we rename `completeAngle` to just startAngle and assign it `-completeAngle`

Didn't catch the comment

Reply:
`CompleteAngle` is confusing to me. Can we do `const startAngle = - (Math.PI / 2 + GaugeComponent.EXTRA_ARC_ANGLE);` ?

Reply:
So, `endAngle =   startAngle + 2 * (inputData.value / inputData.maxValue) * (- startAngle)